### PR TITLE
sidebar feature

### PR DESCRIPTION
**Pull request by rinturejikc**

### CODE REVIEW

**Comment by rinturejikc** [comment]
sidebar part

File: src/App.jsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
-import { Routes, Route } from 'react-router-dom'
+ import { Routes, Route } from 'react-router-dom'
```
Comment (JaseelaAlimahamed) [inline src/App.jsx:1]: nikhil is doing side bar already in user side . please contact him and clarify it

Comment (JaseelaAlimahamed) [reply src/App.jsx:1]: also please donot rrequest to main

Comment (rinturejikc) [reply src/App.jsx:1]: ok

**Pull request closed without merging**